client/blocks/login/error-notice.jsx

```diff
@@ -70,6 +72,27 @@ class ErrorNotice extends Component {
 
 		let message = error.message;
 
+		const { signupUrl } = this.props;
+
+		if ( error.code === 'unknown_user' && signupUrl ) {
+			message = this.props.translate(
+				"Hmm, we can't find a WordPress.com account for that social login. Please double check your information and try again." +
```

Comment:
Would it be better to reuse the string that comes back directly from the API?

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/36432/180864938-3c7206fc-711d-41e9-969c-3f30c746afc1.png">

Reply:
I think that's fine, as long as it's properly translated. We could move part of the existing message to a comment, clarifying what this state is all about.

Reply:
Added a clarifying comment with c97a7cb6182c5c8d98214767b64fca6193ec00f0